python/ray/dag/compiled_dag_node.py

```diff
@@ -1786,75 +1786,25 @@ def _build_execution_schedule(
 
     def _detect_deadlock(self) -> bool:
         """
-        Check whether the DAG will deadlock on NCCL calls.
-        There are no false positives in this deadlock detection,
-        but there may be false negatives for now. For example,
+        TODO (kevin85421): Avoid false negatives.
```

Comment:
Do we have a ticket to track this?

Reply:
open an issue: https://github.com/ray-project/ray/issues/49249